src/community.rs

```diff
@@ -0,0 +1,54 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::{env, AccountId};
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Github {
+    pub repo: String,
+    pub labels: Vec<String>,
+}
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Community {
+    pub name: String,
+    pub description: String,
+    pub image: String,
+    pub thumbnail: String,
+    pub overview: String,
+    pub events: String,
```

Comment:
These are markdown strings, right? Let's also indicate that explicitly through naming: `overview_page_markdown`, `events_page_markdown`

Reply:
Sounds good